include/Contact.php

```diff
@@ -599,6 +594,12 @@ function get_contact($url, $uid = 0, $no_update = false) {
 		$data = $gcontacts[0];
 	}
 
+	// Unable to convert nick@server.tld into http://server.tld/nick
```

Comment:
I guess that we don't need these lines. When the probing fails, the system will return NETWORK_PHANTOM as a network.